docs/applications/radarr.mdx

```diff
@@ -33,64 +34,54 @@ import SystemdTabs from '../snippets/systemdtabs.mdx';
 <SystemdTabs service="radarr"/>
 
 ## Configuration
+The Radarr Team built a detailed [Quick Start Guide](https://wiki.servarr.com/Radarr_Quick_Start_Guide).
+[Radarr's FAQ](https://wiki.servarr.com/Radarr_FAQ) may be of additional use as well.
 
-Out of the box, Radarr comes with very little configuration. Following are some basic tasks to help get your client up and running:
+Out of the box, Radarr comes with no configuration, so you'll need to set it up!
 
-### Add a new show (and your first root directory)
-
-1. At the top, click series.
-2. Click the button `+ Add Movie`
-3. Type the name of the show you want to add
-4. Under `Path`, choose `Add a new path`. Enter the path for the folder you wish to add your Sonarr shows to and click the green check mark to add the directory.
-  - e.g. `/home/<username>/media/Movies`
-5. Change the other settings to your own needs
-6. Click the green + to add the show.
-
-### Connect download clients
-import Conn4Arrs from '../snippets/conn4arr.mdx';
+Some important Swizzin specific information to note:
+### Connect download clients	
+import Conn4Arrs from '../snippets/conn4arr.mdx';	
 
 <Conn4Arrs />
 
-### Add an indexer with Jackett
-Radarr only supports a few trackers out of the box, but the indexers can be expanded by using the application [Jackett](jackett.mdx). To add an indexer with Jackett, use the following steps.
+### Add an indexer with Jackett	
+Radarr only supports a few trackers out of the box, but the indexers can be expanded by using the application [Jackett](jackett.mdx). To add an indexer with Jackett, use the following steps.	
 
-:::note
-By default, when receiving the torznab feed from Jackett, it will need to be modified to be connected with the application. The default link looks like:
-
-```plaintext
-https://<yourhostname.ltd>/jackett/api/v2.0/indexers/yourtracker/results/torznab/
-```
+:::note	
+By default, when receiving the torznab feed from Jackett, it will need to be modified to be connected with the application. The default link looks like:	
 
-We need to edit this link to **remove https** and **add the port number**. Thus, if your port number was `12345`, your edited URL would look like this:
+```plaintext	
+https://<yourhostname.ltd>/jackett/api/v2.0/indexers/yourtracker/results/torznab/	
+```	
```

Comment:
huh what is the difference here? `git` u drunk?

Reply:
there's some reordering that's it?
`git` is drunk

Reply:
wait isn't this section a dupe?

Reply:
it's some formatting it seems 
<img width="1238" alt="image" src="https://user-images.githubusercontent.com/23618693/102909904-f493f180-4479-11eb-9e86-2d8ef097afb2.png">